Feature/docs (#17)
loupeteam/TcCommando: Docs/Overview.md

## Version Information
This documentation is for version `0.01.0` of the TcCommando library.

## Overview

> TcCommando provides a way to abstract and monitor an object's commands in your TwinCAT solution. TcCommando decouples function calls and response monitoring from implementation. This enables remote function calls, multiple instances of function calls with separate status monitoring, and ways to group calls together and manage their status as a group. 

Reviewer: This early in the documentation, a reader may not be familiar with this meaning of "remote" function calls. Can we describe this capability without using our jargon?

> TcCommando provides a way to abstract and monitor an object's commands in your TwinCAT solution. TcCommando decouples function calls and response monitoring from implementation. This enables remote function calls, multiple instances of function calls with separate status monitoring, and ways to group calls together and manage their status as a group. 

Reviewer: ```suggestion
TcCommando provides a way to decouple method calls and response monitoring from a command's implementation. This enables remote method calls, multiple instances of method calls with separate status monitoring, and ways to group calls together and manage their status as a group. 

```
The second sentence is a stronger definition IMO. Also, functions exist outside an object definition, methods specifically belong to an object definition.
